src/app/modules/workspace/workspace.service.ts

```diff
@@ -253,24 +240,24 @@ export const verifyWorkspaceAdmin = (
   return okAsync(true as const)
 }
 
-export const removeFormFromAllWorkspaces = ({
-  formId,
+export const removeFormsFromAllWorkspaces = ({
+  formIds,
   userId,
 }: {
-  formId: string
+  formIds: string[]
   userId: string
 }): ResultAsync<true, DatabaseError> => {
   return ResultAsync.fromPromise(
     WorkspaceModel.removeFormIdsFromAllWorkspaces({
       admin: userId,
-      formIds: [formId],
+      formIds,
     }),
     (error) => {
       logger.error({
         message: 'Database error encountered when archiving form',
         meta: {
           action: 'archiveForm',
```

Comment:
Not from your changes, but in similar spirit as above, should `meta.action` be `removeFormsFromAllWorkspaces` instead?

Reply:
[b734172](https://github.com/opengovsg/FormSG/pull/6754/commits/b734172b81001f24780c9a374ae0ee1db95bdf97) 🙏